docs/weaviate/concepts/vector-quantization.md

```diff
@@ -108,34 +109,63 @@ The size of the training set is configurable. The default is 100,000 objects per
 
 When SQ is enabled, Weaviate boosts recall by over-fetching compressed results. After Weaviate retrieves the compressed results, it compares the original, uncompressed vectors that correspond to the compressed result against the query. The second search is very fast because it only searches a small number of vectors rather than the whole database.
 
+## Rotational quantization
+
+:::caution Technical preview
```

Comment:
I believe we plan to add options other than 8-bit in the next release. Not sure if we want to specify this or not already.

Reply:
Leaving it for now, will update when other bits variants are available.